NET.A.Carting/NET.A.Carting/Controllers/v1/CartsController.cs

```diff
@@ -2,11 +2,15 @@
 using Application.Items.Commands.AddItem;
 using Application.Items.Commands.RemoveItem;
 using Application.Items.Queries;
+using Microsoft.AspNetCore.Authorization;
 using Microsoft.AspNetCore.Mvc;
+using Microsoft.Identity.Web.Resource;
 
 namespace NET.A.Carting.Controllers.v1
 {
     [ApiController]
+    [Authorize(Roles = "Manager")]
```

Comment:
From requirements
_Carting service – all endpoints should be accessible for both roles_